src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_mysql.py

```diff
@@ -256,8 +246,14 @@ def flexible_server_update_custom_func(cmd, instance,
 
 
 def server_delete_func(cmd, client, resource_group_name=None, server_name=None, yes=None):
+    if resource_group_name is None:
+        raise CLIError('Please specify the name of resource group.')
+    if server_name is None:
+        raise CLIError('Please specify the name of server you want to delete.')
```

Comment:
```suggestion
def server_delete_func(cmd, client, resource_group_name, server_name, yes=None):
```
Do you want to make `resource_group_name` and `server_name` required here?

Reply:
You're right. I'll make the change per your comment. Thanks!